### ISSUE TITLE
https://git-scm.com/downloads/win 64 bit standalone binary (Git-2.47.0.2-64-bit.exe) is actually 2.47.0.1.

### ISSUE THREAD

**Issue by JonDeMartini** [git/git-scm.com#1914]
### Which download is failing?

The web page and links are technically correct, but the contents of Git-2.47.0.2-64-bit.exe actually installs 2.47.0.1

### Problem

The executable downloaded from https://github.com/git-for-windows/git/releases/download/v2.47.0.windows.2/Git-2.47.0.2-64-bit.exe actually installs 2.47.0.1 instead of 2.47.0.2 as the name would indicate.

Version 2.47.0.1 has a bug related to fetching fixed in 2.47.0.2. (https://github.com/git-for-windows/git/issues/5199)

### Operating system and browser

Windows 11, Firefox

### Steps to reproduce

Demonstration of problem
1. Uninstall any currently installed version of Git on a Windows machine (test was conducted on Windows 11)
2. Go to https://git-scm.com/downloads/win and download 64-bit Git for Windows Setup under Standalone Installer [64-bit Git for Windows Setup](https://github.com/git-for-windows/git/releases/download/v2.47.0.windows.2/Git-2.47.0.2-64-bit.exe)
3. Install the downloaded executable
4. Execute from Git Bash or other prompt with visibility to git: "git --version --build"
5. Version will include the string "git version 2.47.0.windows.1"

My workaround fix:
6. Uninstall git
7. From powershell get latest git using "winget install --id Git.Git -e --source winget"
8. "git --version --build" will now show "git version 2.47.0.windows.2"

**Comment by dscho** [comment]
Are you sure about that? I just downloaded that file, and started it. Already the title says `.2`:

![Image](https://github.com/user-attachments/assets/1bb27d6a-1000-41f6-ba7d-8d067a581ebe)

The website it opens after successful installation also says `(2)`:

![Image](https://github.com/user-attachments/assets/e5dd1eea-c734-4dff-910b-aef94ea985c9)

Finally, running `git version` in Git Bash agrees, too:

![Image](https://github.com/user-attachments/assets/b7c6c586-8388-4d55-8d8c-d149137289c1)

In short: My local attempts at reproducing the issue refute that there is a problem...

**Comment by dscho** [comment]
@JonDeMartini I have a download of the same file from Oct 31 and it has the expected SHA-256 sum 83c618dcc50b63f0f7afc86a7a125169bd59559ff680683f1d915c45e05ff4cc (compare with the text shown at https://github.com/git-for-windows/git/releases/tag/v2.47.0.windows.2):

![Image](https://github.com/user-attachments/assets/bfc2cabd-b180-450b-a769-f1d2e890c9bf)
